src/modules/75-cd/components/PipelineSteps/CloudFormation/ConnectorStep/CFRemoteWizard.tsx

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useState } from 'react'
+import cx from 'classnames'
+import { useParams } from 'react-router-dom'
+import { Button, ButtonVariation, StepWizard } from '@harness/uicore'
+import { Classes, Dialog, IDialogProps } from '@blueprintjs/core'
+import { useStrings } from 'framework/strings'
+import type { ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import ConnectorDetailsStep from '@connectors/components/CreateConnector/commonSteps/ConnectorDetailsStep'
+import VerifyOutOfClusterDelegate from '@connectors/common/VerifyOutOfClusterDelegate/VerifyOutOfClusterDelegate'
+import GitDetailsStep from '@connectors/components/CreateConnector/commonSteps/GitDetailsStep'
+import DelegateSelectorStep from '@connectors/components/CreateConnector/commonSteps/DelegateSelectorStep/DelegateSelectorStep'
+import {
+  ConnectorMap,
+  getBuildPayload,
+  ConnectorTypes,
+  GetNewConnector,
+  ConnectorStepTitle
+} from '../CloudFormationHelper'
+import ConnectorStepOne from './ConnectorStepOne'
+import { CFFileStore } from './CFFileStorePath'
+import css from '../CloudFormation.module.scss'
+
+const DIALOG_PROPS = {
+  usePortal: true,
+  autoFocus: true,
+  canEscapeKeyClose: true,
+  canOutsideClickClose: true,
+  enforceFocus: false,
+  style: {
+    width: 1175,
+    minHeight: 640,
+    borderLeft: 0,
+    paddingBottom: 0,
+    position: 'relative',
+    overflow: 'hidden'
+  }
+}
+interface Path {
+  [key: string]: string
+}
+
+interface StepChangeData<SharedObject> {
+  prevStep: number
+  nextStep: number
+  prevStepData: SharedObject
+}
+
+const CFRemoteWizard = ({
+  readonly,
+  allowableTypes,
+  showModal,
+  onClose,
+  isParam = false,
+  initialValues,
+  setFieldValue,
+  index,
+  regions
+}: any) => {
+  const { getString } = useStrings()
+  const { accountId, projectIdentifier, orgIdentifier } = useParams<ProjectPathProps>()
+  const [isEditMode, setIsEditMode] = useState(false)
+  const [showNewConnector, setShowNewConnector] = useState(false)
+  const [selectedConnector, setSelectedConnector] = useState('')
+  const connectorStepTitle = getString(ConnectorStepTitle(isParam))
+  const fileStoreTitle = getString(
+    isParam ? 'cd.cloudFormation.parameterFileDetails' : 'cd.cloudFormation.templateFileStore'
+  )
+  const onStepChange = (arg: StepChangeData<any>): void => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
  const onStepChange = (arg: StepChangeData<unknown>): void => {
```